pkg/controllers/scheduler/framework/util.go

```diff
@@ -59,7 +63,7 @@ const (
 
 // TODO(feature), make the RequestedRatioResources configable
 
-var DefaultRequestedRatioResources = ResourceToWeightMap{corev1.ResourceMemory: 1, corev1.ResourceCPU: 1}
+var DefaultRequestedRatioResources = ResourceToWeightMap{corev1.ResourceMemory: 1, corev1.ResourceCPU: 1, ResourceGPU: 4}
```

Comment:
Should we need to explain why GPU ratio is 4?

Reply:
Nice suggestion! Added some comments.

Reply:
Could we further explain why GPU is weighted 4? Simply "based on empirical experience" may not convince critical readers.